controllers/osccluster_net_controller.go

```diff
@@ -58,13 +56,15 @@ func reconcileNet(ctx context.Context, clusterScope *scope.ClusterScope, netsvc
 	if netSpec.ResourceId != "" {
 		netRef.ResourceMap[netName] = netSpec.ResourceId
 		netId := netSpec.ResourceId
+		clusterScope.Info("Check if the desired net exist")
```

Comment:
Maybe add more information ? name of the net ?

Reply:
Ok done